### PR TITLE
Expose metrics-collector scrape limit size in CRD

### PR DESCRIPTION
**Pull request by saswatamcode**
Expose the scrape size limit as a CRD option.

### CODE REVIEW

**Review by thibaultmg — APPROVED** [review]
lgtm

File: operators/endpointmetrics/pkg/collector/metrics_collector.go [hunk 1]
```diff
@@ -822,7 +822,7 @@ func (m *MetricsCollector) getCommands(isUSW bool, deployParams *deploymentParam
 		"--to-upload-key=/tlscerts/certs/tls.key",
 		"--interval=" + interval,
 		"--evaluate-interval=" + evaluateInterval,
-		"--limit-bytes=" + strconv.Itoa(limitBytes),
+		"--limit-bytes=" + strconv.Itoa(m.ObsAddon.Spec.ScrapeSizeLimitBytes),
```
[the remark this answers is not in the document]
Comment (saswatamcode) [reply operators/endpointmetrics/pkg/collector/metrics_collector.go:825]: Ah right missed it

**Review by philipgough — APPROVED** [review]
/lgtm

**Review by coleenquadros — APPROVED** [review]

**Review by philipgough — APPROVED** [review]
/lgtm

**Pull request merged**